pwa/src/pages/test.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from "react";
```

Comment:
We do not destruct `react` imports in our codebase. This works fine, but it's just not how we do it. And for consistency's sake, I'd recommend updating this to:

```
import * as React from "react";
```